modules/ocf_kubernetes/manifests/loadbalancer.pp

```diff
@@ -0,0 +1,38 @@
+class ocf_kubernetes::loadbalancer {
+  include ::haproxy
+
+  $kubernetes_workers_ipv4 = lookup('kubernetes::workers_ipv4')
+  $kubernetes_worker_nodes = lookup('kubernetes::worker_nodes')
+
+  haproxy::frontend { 'kubernetes-frontend':
+    mode    => 'http',
+    bind    => {
+      "${::ipaddress}:80"   => [],
```

Comment:
Can this just bind to all IPs instead so it listens on IPv6 too? Looks like using `0.0.0.0` or just leaving it empty makes it listen on all which I think is what you want here.